db/migrate/20230215085312_migrate_message_body_to_action_text.rb

```diff
@@ -0,0 +1,10 @@
+class MigrateMessageBodyToActionText < ActiveRecord::Migration[7.0]
+  include ActionView::Helpers::TextHelper
+  def change
+    rename_column :messages, :body, :body_old
+    Message.all.each do |message|
+      message.update_attribute(:body, simple_format(message.body_old))
+    end
+    remove_column :messages, :body_old, :text
```

Comment:
are you sure that `remove_column` takes three arguments?

Reply:
yes, it's optional, but will be used in context of the `change` method to make the operation reversible.
https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/remove_column

I realized that the migration isn't reversible at all and I'll rewrite this bit